src/components/country/country.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams, Link } from 'react-router-dom';
+import { countryInfo } from '../../Redux/countries/countries';
+import microbe from '../../assets/iconmicrobe.png';
+import leftArrow from '../../assets/leftarrow.png';
+import './country.css';
+
+export default function Info() {
+  const countryDetails = useSelector((state) => state.details);
+  const { countryId } = useParams();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(countryInfo(countryId));
+  });
+  return (
+    <div className="country-details-container">
+      <div className="detail-container">
+        <Link to="/">
+          <img
+            className="back-left-img"
+            src={leftArrow}
+            width={30}
+            height={30}
+            alt="leftarrow"
+          />
+        </Link>
+        <div className="img-container">
+          <img src={countryDetails.country_flag} alt="country-flag" />
+        </div>
+        <h2>{countryDetails.country_name}</h2>
+        <ul className="country-details">
+          <li>
+            The country covid19 cases are :
+            {' '}
+            <span className="details">{countryDetails.country_cases}</span>
+          </li>
+          <li>
+            The country number of deaths :
+            {' '}
+            <span className="details">{countryDetails.country_death}</span>
+          </li>
+          <li>
+            The country belongs to
+            {' '}
+            <span className="details">{countryDetails.country_continent}</span>
```

Comment:
- [ ] your design is good but one of the requirements is to follow the design provided and also your detail page is not responsive in mobile screen so please you should follow the layout of the design and also try to make it responsive 
- [ ] 
![image](https://user-images.githubusercontent.com/61790587/175401712-af45ae7a-bffe-4085-9f8c-23327b8a6306.png)

![image](https://user-images.githubusercontent.com/61790587/175401528-4884dbf9-33f5-49c1-99e5-37c671a1c634.png)
![image](https://user-images.githubusercontent.com/61790587/175401767-31a99c88-e913-4498-b178-a02d27fb7bbc.png)
- Design 
![image](https://user-images.githubusercontent.com/61790587/175402049-42de8530-b24c-4e91-a2f4-7daf6363c703.png)